kamyaD/Front-end-test-React
# #TDL-001: Create new list and filter todos

### What does this PR do?

- filter todos between "completed", "active" or all
- Create new lists and show all of the lists of todos


### Description of tasks to be completed?

1) Create a new list button
2) Add ability to render list headers, and on click, it expands a container that contains a list of todos and an input.
3) Add a filter todo button/dropdown under each todo input


### How should this   be manually tested

After cloning the repo, checkout to `ft-filter-todo-001` branch  
-  RUN `npm install ` to install dependencies
-  RUN `npm start`  to start the application
-  click `Create new list ` to add a new list
-  click `To do list 0`  to reveal  the default list
- click `Filter todos` to reveal `Complete`, `Active` and `all`, select each to filter 
- Add a new todo under `Add new todo` section  
- click on a check box on revealed list to mark it as completed


### Any Background context you want to provide?
NA

##  Screenshots to the feature ?
##### All lists
![image](https://user-images.githubusercontent.com/33648833/66759687-64d3da00-eea9-11e9-880a-ae729b6c7c07.png)

##### completed list
![image](https://user-images.githubusercontent.com/33648833/66813851-39e79580-ef3e-11e9-9eca-af618e2be5cc.png)


##### Active list
![image](https://user-images.githubusercontent.com/33648833/66814143-c5f9bd00-ef3e-11e9-99ea-d6054273d3ea.png)
